django/santropolFeast/member/models.py

```diff
@@ -36,11 +36,11 @@ class Meta:
     gender = models.CharField(
         max_length=1,
         choices=(
-            ('H', 'Homme'),
-            ('F', 'Femme'),
-            ('U', 'Inconnu'),
+            ('M', 'Male'),
```

Comment:
The 'M' is ok but for the rest we need to have identifiants and not the english version!

So you need to :
 - Delete the caps of the word
 - Add the translation function on each ID